microsoft-edge/progressive-web-apps/how-to/index.md

```diff
@@ -6,179 +6,192 @@ ms.author: msedgedevrel
-
 
 <!-- ====================================================================== -->
-## Step 2 - Create your app start page
+## Front-end code files (UI code)
 
-So far, there is no content available on your web server. Start by creating the first page that users will see when they access your temperature converter app.
+A Progressive Web App (PWA) has front-end code files that are sent by the web server to the browser on the local device.
 
-1. Open Visual Studio Code, select **File** > **Open Folder** and then select the `MySamplePWA` directory you created in the previous step.
+Front-end code is the resources needed for the app to be installed on the user's device, such as HTML, CSS, and JavaScript code.
 
-1. Create a new file in the project by pressing **Ctrl+N**, add the following content, and save the file as `index.html`:
+A Progressive Web App (PWA) includes the following front-end code files:
 
-   ```html
-   <!DOCTYPE html>
-   <html lang="en-US" dir="ltr">
-     <head>
-       <meta charset="UTF-8" />
-       <meta name="viewport" content="width=device-width,initial-scale=1" />
-       <link rel="shortcut icon" href="https://c.s-microsoft.com/favicon.ico?v2" />
-       <title>Temperature converter</title>
-     </head>
-     <body>
-       <h1>Temperature converter</h1>
-     </body>
-   </html>
-   ```
+* An HTML file to describe the content in your app, such as the text, images, text fields, or buttons that appear in the user interface.
 
-1. Go to `http://localhost:8080` to view your app:
+* A CSS file to organize the HTML content in a layout, and provide styles to elements.
 
-   ![Running your new PWA on localhost](./index-images/sample-pwa-app.png)
+* A JavaScript file to add user interactions to your user interface.
 
-The app runs in the browser for now, and can't be installed. To make the app installable, the app needs a web app manifest.
+* A JSON manifest file to describe your app to the host operating system.
 
+The front-end code for a Progressive Web App (PWA) runs by using the device's web browser.  The browser's own UI is not visible when the app is run in a standalone window, which is a streamlined browser window with minimal browser UI controls.
 
-<!-- ====================================================================== -->
-## Step 3 - Create a web app manifest
+The sample Progressive Web App (PWA) consists of the following files, in the `/Demos/pwa-getting-started/` source code directory:
 
-A _web app manifest_ is a JSON file containing metadata about your app, such as its name, description, icons, and the various operating system features it uses.
+| File | Description | Section below |
+|---|---|---|
+| `converter.css` | The Cascading Style Sheets (CSS) file defines styling for the webpage of the app; the layout styling of each UI control in the main page of the app. | [The JavaScript app logic file (`converter.js`)](#the-css-style-sheet-file-convertercss) |
```

Comment:
sort rows same as h2 order (not directory alpha order w/ Readme first like at GH)